### PR TITLE
chore(#8264): restructured add-new-district.wdio-spec.js test

### PR DESCRIPTION
**Pull request by tatilepizs**
# Description

Re-structured the `add-new-district.wdio-spec.js` test to make it more stable.

medic/cht-core#8264

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs

If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8264-flaky-test-add-new-district/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8264-flaky-test-add-new-district/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8264-flaky-test-add-new-district/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Review by ngaruko — APPROVED** [review]
Looks good. Tested a few times and it seems to be solid.
Thanks

File: tests/e2e/default/contacts/add-new-district.wdio-spec.js [hunk 1]
```diff
@@ -1,83 +1,78 @@
...
+const placeFactory = require('@factories/cht/contacts/place');
+const personFactory = require('@factories/cht/contacts/person');
+const commonPage = require('@page-objects/default/common/common.wdio.page');
+const contactPage = require('@page-objects/default/contacts/contacts.wdio.page');
 const loginPage = require('@page-objects/default/login/login.wdio.page');
 
+
 describe('Add new district tests : ', () => {
   before(async () => await loginPage.cookieLogin());
   afterEach(() => sentinelUtils.waitForSentinel());
```
Comment (lorerod) [inline tests/e2e/default/contacts/add-new-district.wdio-spec.js:12]: Please change this so retries are possible.
```suggestion
  afterEach(async () => {
    sentinelUtils.waitForSentinel();
    await utils.revertDb([/^form:/], true);
  });
```

**Review by lorerod — CHANGES_REQUESTED** [review]
This looks great @tatilepizs! Just a change request to make retries possible.

**Review by ralfudx — APPROVED** [review]
Looks good to me... although not entirely sure about the hardcoded test variables but will be nice to have further discussions about them in future

**Review by lorerod — APPROVED** [review]
Gracias @tatilepizs !

**Pull request merged**